gateway/1.5/modules/ROOT/pages/flex-sizing-guide.adoc

```diff
@@ -0,0 +1,65 @@
+= Reviewing Resource Sizing 
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+:table-caption!:
+
+Before getting started with Flex Gateway, analyze and choose your configuration to obtain the best performance and auto-scaling rate.
+
+When assigning resources for a replica node, consider:
+
+* Number of APIs
+* Amount of memory the node needs
+* Processing power for the node
+* Number of policies in each API
+* Amount of traffic each node handles
+* Amount of time the node takes to answer the request (latency)
+* Type of traffic the node handles, and within the traffic:
+** The number of requests per second the node handles
+** Size of each request (bytes)
+
+== Choose Appropriate Resources
+
+The following table guides you to know the resources you must allocate to your replica node according to your environment:
+
+[options="header"]
+|====
+| Node Size | CPU | RAM 
+| Small | 1-2 cores | 2-4 GB
+| Medium | 2-4 cores | 4-8 GB
+| Large | 8-16 cores | 16-32 GB	
+|====
+
+NOTE: Nodes belonging to a cluster must be of the same size.
+
+== Choose Your Cluster Size
+
+The following guiding table helps you choose the appropriate cluster size:
```

Comment:
"...appropriate node and cluster size..."

Reply:
OK! Thanks!